<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>High Tower Text</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(85, 85, 127);
</string>
  </property>
  <widget class="QPushButton" name="LoginInbtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.999316000000000" starty="0.353000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>57</red>
           <green>166</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>163</red>
           <green>163</green>
           <blue>163</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>173</red>
           <green>173</green>
           <blue>173</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>163</red>
           <green>163</green>
           <blue>163</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>173</red>
           <green>173</green>
           <blue>173</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>163</red>
           <green>163</green>
           <blue>163</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>173</red>
           <green>173</green>
           <blue>173</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="statusTip">
    <string>登录</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-radius:10px;
	color:white;
	font-size:17px;
	box-shadow: 1px 2px 3px 4px;
	background:qlineargradient(spread:pad, x1:0.999316, y1:0.353, x2:1, y2:0, stop:1 rgba(57, 166, 255, 255));
}
QPushButton:hover
{
	border-radius:5px;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(6, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}
QPushButton:disabled
{
	border-radius:10px;
	color:black;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(163, 163, 163, 255), stop:1 rgba(173, 173, 173, 255));
}
QPushButton:press
{
	border-radius:2px;
	background-color:transparent;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(21, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
   </property>
   <property name="text">
    <string>进入</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Enter</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string>输入密码</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>用户名</string>
   </property>
   <property name="statusTip">
    <string>输入用户名</string>
   </property>
   <property name="accessibleName">
    <string>用户名</string>
   </property>
   <property name="accessibleDescription">
    <string>用户名</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;


}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="createUser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(222, 222, 222);
	border: 0px;
	background-color:transparent;
}
QPushButton:hover
{
	color: rgb(0, 170, 255);
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>创建用户</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="userIcon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/usernameWHITE</normaloff>:/icon1/usernameWHITE</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pswdIcon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/pswdWHITE</normaloff>:/icon1/pswdWHITE</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Help">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
}
QPushButton:hover
{
	border: 0px;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/helpWHITE</normaloff>:/icon1/helpWHITE</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="userLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 2px solid white; 
background-color:transparent;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="pawdLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 2px solid white; 
background-color:transparent;
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
}
QPushButton:hover
{
	background: qlineargradient(spread:pad, x1:0.00526316, y1:0.006, x2:1, y2:1, stop:0 rgba(8, 8, 127, 255), stop:1 rgba(85, 85, 127, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/settingWHITE</normaloff>:/icon1/settingWHITE</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setting_2">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
}
QPushButton:hover
{
	background: qlineargradient(spread:pad, x1:0.00526316, y1:0.006, x2:1, y2:1, stop:0 rgba(8, 8, 127, 255), stop:1 rgba(85, 85, 127, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/minWHITE</normaloff>:/icon1/minWHITE</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;

}
QPushButton:hover
{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 85, 0, 255), stop:1 rgba(255, 0, 255, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/closeWHITE</normaloff>:/icon1/closeWHITE</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
